src/fetch_receipts.py

```diff
@@ -41,28 +47,31 @@ def all(self):
         }
         for dataset in self.datasets():
             data = pd.read_csv(dataset, parse_dates=[16], dtype=dtype)
-            yield from(Receipt(row) for row in data.itertuples())
+            yield from(Receipt(row, self.target) for row in data.itertuples() if not pd.isnull(row.document_id))
```

Comment:
If we keep `target` here, you're changing the output of this class without updating the documentation (__docstring__). Can you update that, please?